client/landing/gutenboarding/onboarding-block/designs/index.tsx

```diff
@@ -38,8 +38,11 @@ const Header: React.FunctionComponent = () => {
 	const { __ } = useI18n();
 	const locale = useLocale();
 	const isAnchorFmSignup = useIsAnchorFm();
+	const { isEnrollingInFseBeta } = useSelect( ( select ) =>
```

Comment:
The `useSelect` callback is just returning a bool here

```suggestion
	const isEnrollingInFseBeta = useSelect( ( select ) =>
```

Reply:
Thank you!